doc/src/manual/environment-variables.md

```diff
@@ -107,6 +107,16 @@ Suppose the value of `$JULIA_PKGRESOLVE_ACCURACY` is `n`. Then
 *   the number of iterations between decimation steps is `10*n`, and
 *   at decimation steps, at most one in every `20*n` packages is decimated.
 
+### `JULIA_DEPOT_HOME`
+
+A directory path that points to the default location for storage of Julia packages. If `$JULIA_DEPOT_HOME` is not set, then `REPL.DEPOT_PATH` defaults to
```

Comment:
`JULIA_DEPOT_HOME` -> `JULIA_DEPOT_PATH` and `REPL.DEPOT_PATH` -> `Base.DEPOT_PATH`